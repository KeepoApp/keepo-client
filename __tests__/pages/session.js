import React from 'react';
import renderer from 'react-test-renderer';

import Session from '../../app/containers/pages/session/index';

describe('Session page', () => {
  it('should render correctly', () => {
    const component = renderer.create(
      <Session />,
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
