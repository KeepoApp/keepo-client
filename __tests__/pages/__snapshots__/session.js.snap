// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Session page should render correctly 1`] = `
<h1>
  Hallo wereld!
</h1>
`;
